datalake/src/main/java/com/sequenceiq/datalake/flow/delete/SdxDeleteActions.java

```diff
@@ -189,6 +190,13 @@ protected void doExecute(SdxContext context, SdxDeletionFailedEvent payload, Map
                             payload.getResourceId());
                     metricService.incrementMetricCounter(MetricType.SDX_DELETION_FAILED, sdxCluster);
                     eventSenderService.notifyEvent(context, ResourceEvent.SDX_CLUSTER_DELETION_FAILED);
+
+                    if (sdxCluster.isDetached()) {
+                        eventSenderService.sendEventAndNotification(
+                                sdxCluster, context.getFlowTriggerUserCrn(), ResourceEvent.SDX_DETACHED_CLUSTER_DELETION_FAILED,
```

Comment:
Can we retrigger this operation to delete only the detached DL?

Reply:
While I do think the concept of retrying would be nice, I don't think it makes much sense to introduce that here. It would be kind of odd to retry deletion in case of failure _only_ for detached DLs. While I'm open to the idea of retrying deletions in general, I think that should be part of a larger more complex change.

In the meanwhile I will rephrase my event message a little to state that retrying via the UI is most likely the best option since that would be more ideal than the CLI.